proto/bff/models/v1/models.proto

```diff
@@ -138,10 +138,12 @@ enum AttentionSeverity {
 // AttentionAction is used to indicate the action that should be taken by the user in
 // response to an attention message.
 enum AttentionAction {
-    START_REGISTRATION = 0;
-    COMPLETE_REGISTRATION = 1;
-    SUBMIT_TESTNET = 2;
-    SUBMIT_MAINNET = 3;
-    RENEW_CERTIFICATE = 4;
-    CONTACT_SUPPORT = 5;
+    NO_ACTION = 0;
```

Comment:
Guessing we're intentionally using the 0-val of the enum to do nothing by default if no val is passed in?

Reply:
It's not really a default value, but we do need a value to represent the "no action required" messages, such as when a registration is approved.